mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -489,7 +489,7 @@ CLASS aif net/minecraft/entity/Entity
 	METHOD j getCameraPosVec (F)Lcrv;
 		ARG 1 tickDelta
 	METHOD j allowsPermissionLevel (I)Z
-	METHOD j onBubbleColumnEjection (Z)V
+	METHOD j onBubbleColumnSurfaceCollision (Z)V
```

Comment:
Actually also fixes `suface` instead of `surface`, compared to main branch.
It is the better name though, since it's also used for bubble columns that drag, so it can also suck entities in, with a higher downwards velocity than usually.